contents.json

```diff
@@ -1818,6 +1818,11 @@
       "category": "images",
       "description": "Implementation of [fastimage](https://pypi.org/project/fastimage/0.2.1/) - supports PNG, GIF, and JPEG.",
       "homepage": "https://github.com/kaishin/ImageScout"
+    }, {
+      "title": "ImgixSwift",
+      "category": "images",
+      "description": "Easily update image urls to be fast and responsive.",
+      "homepage": "https://docs.imgix.com/libraries/imgix-swift"
```

Comment:
Hi @frederickfogerty, I'm not sure about this URL https://docs.imgix.com/libraries/imgix-swift. Could you replace it to the git repository https://github.com/imgix/imgix-swift.git?

Reply:
Sure thing, I'll do that now 👍🏻